website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -84,15 +84,17 @@ $ curl \
 {
   "data": {
     "cas_required": false,
-    "max_versions": 0,
-    "delete_version_after": "3h25m19s"
+    "delete_version_after": "3h25m19s",
+    "max_versions": 0
   }
 }
 ```
 
 ## Read Secret Version
 
-This endpoint retrieves the secret at the specified location.
+This endpoint retrieves the secret at the specified location. Metadata returned
+here is version specific metadata. Not to be confused with the Metadata set on the
+metadata endpoint.
```

Comment:
This is a good distinction, but the sentences don't read smoothly to me. A suggestion:

> This endpoint retrieves the secret at the specified location. The metadata returned here (`created_time`, `deletion_time`, `destroyed`, and `version`) is version-specific. It should not be confused with the response from the metadata endpoint. 

I might also suggest linking "the metadata endpoint" to that section in the documentation.

Reply:
Perfect and will do!